packages/astro/package.json

```diff
@@ -167,7 +167,9 @@
     "astro-scripts": "workspace:*",
     "chai": "^4.3.6",
     "cheerio": "^1.0.0-rc.11",
+    "mdast-util-to-string": "^3.1.0",
     "mocha": "^9.2.2",
+    "reading-time": "^1.5.0",
```

Comment:
both deps can be removed?

Reply:
Oh, these are dev dependencies for unit tests! Shouldn't reach the end user

Reply:
Can they be added to the fixture instead?

Reply:
I'm applying them from the config within the unit test actually, so I don't need a separate fixture. Sounds like we'd prefer that approach though. I'll refactor!